CHANGELOG-v3.adoc

```diff
@@ -1,5 +1,28 @@
 # Change Log
 
+==  - 3.21.3 (2023-07-07)
```

Comment:
```suggestion
==  AM - 3.21.3 (2023-07-07)
```

Maybe there is a bug to fix in the zx script

Reply:
Yeap. It seems the generated changelog does not include `AM` as the older ones.

My only question is, why are we adding `AM` in each line of what it's already AM's changelog?
Can that be something else?

Reply:
@remisultan 😉 
https://github.com/gravitee-io/gravitee-access-management/pull/2749

Reply:
✅